lib/client/ruby/tunnel-vision.rb

```diff
@@ -26,42 +26,62 @@ class TunnelVision < Thor
 
   class InvalidArgument < ::StandardError; end
 
-  CONFIG_URL = 'https://raw.githubusercontent.com/abhchand/tunnel-vision/master/roles/tunnel-server/vars/main.yml'.freeze
-  TUNNEL_URL = 'exec@pipe.cr-tunnel.xyz'.freeze
+  TUNNEL_HOST = 'pipe.cr-tunnel.xyz'.freeze
+  TUNNEL_USER = 'exec'.freeze
+  VERSION = '0.1.0'
 
   method_option :user, required: true, aliases: '-u', desc: 'User for tunnel hostname'
+  method_option(
+    :application,
+    required: false,
+    default: 'callrail',
+    aliases: '-a',
+    desc: 'Name of your target application'
+  )
```

Comment:
Updating the client script to take a `--application` parameter (which defaults to `callrail`)

This way you can start a separate tunnel for `swappy`, `rowdy`, etc...